src/styles/theme/themes/light.ts

```diff
@@ -87,6 +87,7 @@ const lightTheme = {
     starDefaultBG: 'rgb(254, 228, 94)',
     mapAttributionText: colors.black,
     white: colors.white,
+    progressBarBackground: colors.productLight400,
```

Comment:
Thoughts on using `border` from above here?

Reply:
I think that `progressBarBackground` is similiar enough in use case to the `border` color, I'll remove the redundant definition and replace current uses with `border`.